src/Microsoft.Identity.Client/Internal/Broker/NullBroker.cs

```diff
@@ -0,0 +1,48 @@
+﻿// ------------------------------------------------------------------------------
+// 
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+// 
+// This code is licensed under the MIT License.
+// 
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+// 
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+// 
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+// 
+// ------------------------------------------------------------------------------
+
+using Microsoft.Identity.Client.Core;
+using Microsoft.Identity.Client.OAuth2;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Client.Internal.Broker
+{
+    /// <summary>
+    /// For platforms that do not support a broker (net desktop, net core, UWP, netstandard)
+    /// </summary>
+    internal class NullBroker : IBroker
+    {
+        public bool CanInvokeBroker => false;
+
+        public Task<MsalTokenResponse> AcquireTokenUsingBrokerAsync(Dictionary<string, string> brokerPayload, IServiceBundle serviceBundle)
+        {
+            throw new NotImplementedException();
```

Comment:
>NotImplementedException [](start = 22, length = 23)

not sure, however should this be a PlatformNotImplementedException ?

Reply:
yes...thank you. will fix.

---
In reply to: [255263487](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/874#discussion_r255263487) [](ancestors = 255263487)